service/controller/v5/resource_set.go

```diff
@@ -167,6 +168,19 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 		}
 	}
 
+	var encryptionkeyResource controller.Resource
+	{
+		c := encryptionkey.Config{
+			K8sClient: config.K8sClient,
+			Logger:    config.Logger,
```

Comment:
For the ManagedBy label you'll need to add the projectName here but its useful to have anyway.